### ISSUE TITLE
Research how one might respect the soft delete in vld files.

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#1311]
@geoffj-FUG has requested that
Deleted fields should not display. There will be a lot of them in the early vld files as the proofreading programme used to create them (all soft deletes). Once that app died there were not many of them as hard deletes in spreadsheets became the norm.

At present both FC1 and FC2 retrieve and display records that contain the deleted flag contained  in a vld record.

This story processes to investigate if and how it might be possible to locate and implement code changes so that they are no longer displayed or include the display of the soft delete flag.

E0 U1 S3 C6:  P10

**Comment by geoffj-FUG** [comment]
My last hypothesis was that the D flags were some sort of historical leftover from FC1 where these flags existed? The original hypothesis was that the Ds were the control flags from valdrev. But there is no obvious trigger for these flags to occur.
I have now re-validated over 7000 entries from a vld file which included about 200 odd entries which were marked as having a record Deleted flag.. What I discovered was that whenever there was a validators choice added there was a D flag associated with it. Where there were various entries that would cause Warnings in valdrev there was a D flag. associated with it.
There were some records without D flags but these were Warnings in FC2, not FC1. There were also entries without D flags which we now display with a note.
It is definite that the D flag is not a record Deleted flag. The note record marked as deleted can be removed from the vld download routine.
Secondly, on the balance of probabilities, the D flag was associated with the display in the original FC1 screen search display. I have found no other explanation for it, nor any reason why we need to be aware of it.
I am highly confident that we can ignore this flag and that it has no affect on the results of a researchers search.
If this is accepted then this story can be closed.
Geoff
